Tensile/Components/LocalRead.py

```diff
@@ -250,12 +266,13 @@ def __call__(self, writer, bufferIdx, iui, epsi, tP):
                           if (kernel["GlobalLoadVectorWidth%s"%tc] * tP["bpe"] == 8):
                             # dword_x2 case
                             # (bit2<<3) | (bit3 >>1) | (bit4>>1) | (bit5>>1)
-                            newVal = (bit2<<3) | (bit3 >>1) | (bit4>>1) | (bit5>>1)
+                            offset_val = offset_val & (~0x24)
+                            newVal = (bit2<<3) | (bit5>>1)
```

Comment:
Why you changed this part?
Without shifting bit3,4 >> 1, 
bit5>>1 will overwrite bit 4 and I did not expect that was correct.